doc/ref_arch/kubernetes/chapters/chapter02.md

```diff
@@ -38,30 +21,30 @@ To ensure alignment with the infrastructure profile catalogue, the following req
 - Those relating to Cloud Infrastructure Management
 - Those relating to Cloud Infrastructure Security
 
-### 2.2.1 Cloud Infrastructure Software Profile Capabilities
+### Cloud Infrastructure Software Profile Capabilities
 
 | Reference Model Section | Reference  | Description | Requirement for Basic Profile | Requirement for High-Performance Profile| Specification Reference |
 |---|---|---|---|---|---|
 | [4.1.2](../../../ref_model/chapters/chapter04.md#412-exposed-infrastructure-capabilities) | e.cap.001 | Max number of vCPU that can be assigned to a single Pod by the Cloud Infrastructure | At least 16  | At least 16  | [ra2.ch.011](chapter04.md#42-kubernetes-node)|
```

Comment:
```suggestion
| [4.1.2](../../../ref_model/chapters/chapter04.md#412-exposed-infrastructure-capabilities) | e.cap.001 | Max number of vCPU that can be assigned to a single Pod by the Cloud Infrastructure | At least 16  | At least 16  | [ra2.ch.011](chapter04.md#kubernetes-node)|
```

Reply:
I've fixed all occurrences in 2a099bd

Reply:
> @karinesevilla @pgoyal01 the intention of the documentation update is to leverage automation for things like header numbering, tables of contents etc. We also have to be conscious that we have 1 input format (currently GitHub-flavored-markdown, in the future maybe rst) and 2 output formats: docx for the GSMA document and html on read-the-docs. The rendering on GitHub is an additional output, yet it is not the output format of the release. Unfortunately, the markdown flavor on GitHub, which is typically used by many on a daily basis, does not allow for much automation.
> 
> So, there is a tradeoff here between the ease of generating two release output formats and the development workflow on GitHub. Not sure if @CsatariGergely schedules another tech discuss meeting on this topic, but feel free to join and we'll discuss.

Sure, I've put it to next weeks [meeting agenda](https://wiki.anuket.io/display/HOME/Anuket+Weekly+Technical+Discussions).